src/components/03-components/03-buttons/buttons.njk

```diff
@@ -2,53 +2,70 @@
   <div class="bg-base-darkest padding-1" style="max-width: fit-content">
   {% endif -%}
 
-  <button class="usa-button {{ classes }}">Default</button>
```

Comment:
Make buttons reusable/use config file

Reply:
Then update throughout Sam Styles

Reply:
We should take fractal optimization tasks, specially actions across the entire codebase in separate tickets/branches so work can be rolled back independently if needed

Reply:
Dave said tickets included optimization during sprint planning